### PR TITLE
Sideways lava and sandwich lava

### PR DESCRIPTION
**Pull request by max4805**
Closes part of #51.

Here is lava coming either from the left of the screen, from the right, or both ("sideways sandwich lava" ~~aka "hot-dog lava"~~). Kind of a mashup between RisingLava and SandwichLava in vanilla.

Again, lots of code pulled directly from vanilla (LavaRect), but tweaked quite a bit to have bubbles go sideways.

### CODE REVIEW

**Comment by max4805** [comment]
Note to self: the way lava blocker triggers are handled here is bad. I should make sideways lava aware of lava blocker triggers instead (collect a list of them in Awake, and collide check them with the player in Update).

**Comment by max4805** [comment]
Just fixed my self-feedback, so I'm done with this now. This is ready to be reviewed and merged if there is no review feedback.

File: Ahorn/entities/sidewaysLava.jl [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module SpringCollab2020SidewaysLava
+
+using ..Ahorn, Maple
+
+@mapdef Entity "SpringCollab2020/SidewaysLava" SidewaysLava(x::Integer, y::Integer, intro::Bool=false, lavaMode::String="LeftToRight", speedMultiplier::Number=1)
+
+const lavaModes = ["LeftToRight", "RightToLeft", "Sandwich"];
```
Comment (Cruor) [inline Ahorn/entities/sidewaysLava.jl]:
```suggestion
const lavaModes = String["LeftToRight", "RightToLeft", "Sandwich"]
```

Comment (Cruor) [reply Ahorn/entities/sidewaysLava.jl]: Small nitpick

**Pull request merged**